examples/README.md

```diff
@@ -6,5 +6,6 @@ Let us know if you find any issue or if you want to contribute and add a new tut
 
 ## Examples and Tutorials
 
+- [Fetching a UnixFS file by CID](./unixfs-file-cid)
```

Comment:
Not sure what order we want for these. Just did alphabetical by title, feel free to change though.